CI/integration_tests/calculators/__test_einstein_distinct_diffusion_coefficients.py

```diff
@@ -58,7 +58,9 @@ def test_eddc_project(traj_files, true_values, tmp_path):
     """Test the EinsteinDistinctDiffusionCoefficients called from the project class"""
     os.chdir(tmp_path)
     project = mds.Project()
-    project.add_experiment("NaCl", data=traj_files[0], timestep=0.002, temperature=1400)
+    project.add_experiment(
+        "NaCl", fname_or_file_processor=traj_files[0], timestep=0.002, temperature=1400
```

Comment:
I think this argument name needs changing. Perhaps `experiment_data`, `simulation_data` or something along these lines.